test/test-raja-perf-suite.cpp

```diff
@@ -0,0 +1,120 @@
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+// Copyright (c) 2017-22, Lawrence Livermore National Security, LLC
+// and RAJA Performance Suite project contributors.
+// See the RAJAPerf/LICENSE file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+
+#include "gtest/gtest.h"
+
+#include "common/Executor.hpp"
+#include "common/KernelBase.hpp"
+
+#include <vector>
+#include <string>
+#include <iostream>
+#include <cmath>
+
+TEST(ShortSuiteTest, Basic)
+{
+
+// Assemble command line args for basic test
+  int argc = 4;
+#if defined(RAJA_ENABLE_HIP)
```

Comment:
@MrBurmark here's what I came up with to disable the failing fused kernel test for hip builds. It's gross and should be checked with new versions. Any thoughts for improvement?

Reply:
I think its reasonable for now, just have to add an issue to make it better later.
Would it be a bit simpler if you used sargv.size() instead of a separate argc?

Reply:
Can you use a newer rocm version, kernel fusion appears to work here with rocm 5.1.

Reply:
I used the argc/argv stuff from the start since it mimics the args to main, MPI_Init, etc. We would have to change the RunParams method that parses the command line arg list if we changed what we pass to the Executor ctor.

Of course, we can update the compiler versions. Let's do in a separate PR. Actually, that would be a good test to see if the RAJA dev guide CI docs I am working on are useful -- have someone see if they can do it following what's there.

Reply:
@MrBurmark rocm 5.1 is available on rznevada. We're not quite there yet with CI testing for RAJA/RAJAPerf. @kab163 is working on it. On corona, the latest available is rocm 5.0.2.

Reply:
We can use the HIP_VERSION to enable/disable the test so it automatically turns on when we can test it https://github.com/LLNL/RAJA/blob/develop/include/RAJA/config.hpp.in#L261.

Reply:
So change the guard to this?

`#if (HIP_VERSION_MAJOR >= 5 && HIP_VERSION_MINOR >= 1)`

Reply:
I think you want something more like this, or the negative of this actually in this case.

```
#if (HIP_VERSION_MAJOR > 5 || (HIP_VERSION_MAJOR == 5 && HIP_VERSION_MINOR >= 1))
```

Reply:
I thought the test fails for rocm 5.0.2?

Reply:
Gotcha.